tests/plugins/test_pull_base_image.py

```diff
@@ -101,7 +115,7 @@ def test_pull_base_image_plugin(parent_registry, df_base, expected, not_expected
     workflow.builder.base_image = ImageName.parse(df_base)
 
     expected = set(expected)
-    expected.add(df_base)
```

Comment:
That's a good point. For some reason when I add that line back in I get test failures... will investigate.